src/lightning-address.ts

```diff
@@ -233,12 +237,13 @@ export default class LightningAddress {
     options: ZapOptions = {},
   ): Promise<SendPaymentResponse> {
     const invoice = this.zapInvoice(args, options);
-    if (!this.webln) {
+    const webln = this.getWebLN()
+    if (!webln) {
```

Comment:
can you also add this check in the boost function above? right now it doesn't check if webln is unavailable.

Also the `// mainly for TS` comment I think can be removed - it's not for TS, it is actually used to give a helpful error if webln does not exist

Reply:
Done!